sympy/physics/mechanics/_pathway.py

```diff
@@ -0,0 +1,248 @@
+        return length
+
+    @property
+    def shortening_velocity(self) -> Expr:
+        """Exact analytical expression for the pathway's shortening velocity."""
+        relative_position = self.attachments[-1].pos_from(self.attachments[0])
+        if not relative_position:
+            return S.Zero
+        t = dynamicsymbols._t  # type: ignore
+        # A reference frame is needed to differentiate ``relative_position`` to
+        # ``relative_velocity`` so choose the first ``ReferenceFrame`` that
+        # ``relative_position`` is defined using.
+        frame = relative_position.args[0][1]
+        relative_velocity = relative_position.diff(t, frame)
+        shortening_velocity = -relative_velocity.dot(relative_position.normalize())
+        return shortening_velocity
+
+    def compute_loads(self, force: Symbol) -> list[tuple[Point, Vector]]:
+        """Loads required by the equations of motion method classes.
+
+        Explanation
+        ===========
+
+        ``KanesMethod`` requires a list of ``Point``-``Vector`` tuples to be
+        passed to the ``loads`` parameters of its ``kanes_equations`` method
+        when constructing the equations of motion. This method acts as a
+        utility to produce the correctly-structred pairs of points and vectors
+        required so that these can be easily concatenated with other items in
+        the list of loads and passed to ``KanesMethod.kanes_equations``. These
+        loads are also in the correct form to also be passed to the other
+        equations of motion method classes, e.g. ``LagrangesMethod``.
+
+        Examples
+        ========
+
+        The below example shows how to generate the loads produced in a linear
+        actuator that produces a contractile force ``F``. First, create a
+        linear actuator between two points separated by the coordinate ``q``
+        in the ``x`` direction of the global frame ``N``.
+
+        >>> from sympy.physics.mechanics import Point, ReferenceFrame
+        >>> from sympy.physics.mechanics._pathway import LinearPathway
+        >>> from sympy.physics.vector import dynamicsymbols
+        >>> q = dynamicsymbols('q')
+        >>> N = ReferenceFrame('N')
+        >>> attachments = (Point('pO'), Point('pI'))
+        >>> attachments[-1].set_pos(attachments[0], q * N.x)
+        >>> linear_pathway = LinearPathway(attachments)
+
+        Now create a symbol ``F`` to describe the magnitude of the
+        (contractile) force that will be produced along the pathway. The list
+        of loads that ``KanesMethod`` requires can be produced by calling the
+        pathway's ``compute_loads`` method with ``F`` passed as the only
+        argument.
+
+        >>> from sympy import Symbol
+        >>> F = Symbol('F')
+        >>> linear_pathway.compute_loads(F)
+        [(pO, F*q(t)/sqrt(q(t)**2)*N.x), (pI, - F*q(t)/sqrt(q(t)**2)*N.x)]
+
+        Parameters
+        ==========
+
+        force : Symbol
```

Comment:
Sorry for yet another comment. Was going through for the last time. I would actually expect `force` to be an instance of `Expr` not just of `Symbol`.

Reply:
Addressed in https://github.com/sympy/sympy/pull/24914/commits/4058172edeff72baf789ee44988fca87ff78a666.